JSDemos/Demos/DateBox/Overview/description.md

```diff
@@ -4,7 +4,16 @@ The [DateBox](/Documentation/ApiReference/UI_Components/dxDateBox/) editor helps
 A value the DateBox displays.
 
 - [type](/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#type)       
-Specifies whether the DateBox allows users to select only date, only time, or both.
+Specifies whether the DateBox allows users to select one of the following types:
+
+  - *"date"*    
+  Users can select the date from the calendar, or they can type in their own values.
+
+  - *"time"*    
+  Users can select from the time slots that lay in range from 12 AM to 11:30 PM with a step of 30 minutes, or they can type in their own values. Specify the [interval](/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#interval) property to set the different time step.
+
+  - *"datetime"*    
+  Users can select both the date from the calendar and the time. The **Today** button is available in the date picker.
```

Comment:
In the demo, I see that the datetime editor shows a calendar and a clock with editors. Can we say something like the following?

`Users can select the date from the calendar and the time from the spin and dropdown editors. They can also click the **Today** button.`

Reply:
Yes, let's change it this way, that's a great suggestion.

Reply:
Thank you.